Assets/_BForBoss/_Character/Scenes/Sandbox.unity

```diff
@@ -1590,6 +1590,63 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 60e034bf0281a1d40a65b8ba612b340f, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+--- !u!1001 &635532793
```

Comment:
Why are there edits in this scene? 

Probably going to have conflicts, since i'm adding in a new scene template and removing the old sandbox one

Reply:
I can remove the changes then!